docker/wordpress/empty-folder.md

```diff
@@ -0,0 +1,3 @@
+This folder is required by docker setup. 
+
+This file is here to ensure the folder is present in a clean clone.
```

Comment:
I've added these placeholder files to ensure that the folders are present in a clean clone of the repo.

If the folders aren't present, `npm run up` / `docker-compose up` can fail with errors like this:

```
ERROR: for woocommerce_payments_wordpress Cannot start service wordpress: error while creating mount source path '/Users/rua/_automattic/transact-dev/wcpay-docker-testing/woocommerce-payments/docker/wordpress': chown /Users/rua/_automattic/transact-dev/wcpay-docker-testing/woocommerce-payments/docker/wordpress: permission denied
```

Reply:
nit: I usually add an empty folder to Git [this way](https://stackoverflow.com/questions/115983/how-can-i-add-a-blank-directory-to-a-git-repository), that is adding `docker/wordpress/.gitignore` with content
```
# Ignore everything in this directory
*
# Except this file
!.gitignore
```
So it's a hidden file and more standard. Description as you have it in empty-folder.md can be set as comment in the `.gitignore`.

Just a suggestion. Your current approach is fine too =)

Reply:
I use `.gitignore` for WCPay server changes. It doesn't have to be but are you OK to make it consistent here too?

Reply:
> I use .gitignore for WCPay server changes. It doesn't have to be but are you OK to make it consistent here too?

Yes please! `.gitignore` is a much better approach than an "empty folder" file. Feel free to make this change @shendy-a8c – thanks for the tip.

Reply:
.. and now I have switched to local `.gitignore`! 🕚  25ecd52376ff9225818bb3ce46447cad386d8985